docs/WritingTests.asciidoc

```diff
@@ -1378,10 +1378,18 @@ openqa-clone-job --from https://openqa.opensuse.org  --host localhost 24 TESTDEB
 Normally the test schedule, that is which test modules should be executed and
 which order, is prescribed by the `main.pm` file within the test distribution.
 Additionally it is possible to exclude certain test modules from execution
-using the os-autoinst test variables `INCLUDE_MODULES` and `EXCLUDE_MODULES` as
-well as define a custom schedule using the test variable `SCHEDULE`.
-Also test modules can be defined and overridden on-the-fly using a
-downloadable asset.
+using the os-autoinst test variables `INCLUDE_MODULES` and `EXCLUDE_MODULES`
+as well as define a custom schedule using the test variable `SCHEDULE`. Also
```

Comment:
```suggestion
as well as defining a custom schedule using the test variable `SCHEDULE`. Also
```

Reply:
I'd leave it because `...exclude.......define....` should match

Reply:
My English grammar parser demands a noun after "as well as …".

Reply:
I rephrased  to avoid that misunderstanding